product_tag_pricelist/models/product_product.py

```diff
@@ -0,0 +1,21 @@
+# Copyright 2023 Manuel Regidor <manuel.regidor@sygel.es>
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+from odoo import _, api, models
+
+
+class ProductProduct(models.Model):
+    _inherit = "product.product"
+
+    @api.onchange("product_tag_ids", "additional_product_tag_ids")
+    def _onchange_tag_ids(self):
+        if self.product_tag_ids:
```

Comment:
it's not multi?

Reply:
I'll review it

Reply:
Its ony one record. Computes are multi and onchanges are one